services/postfix/aliases.nix

```diff
@@ -0,0 +1,500 @@
+  "news" = "root";
+  "nobody" = "/dev/null";
+  "operator" = "root";
+  "pop" = "root";
+  "system" = "root";
+  "toor" = "root";
+  "usenet" = "news";
+  "uucp" = "root";
+  "xten" = "root";
+  "postfix" = "root";
+  "abuse" = "rb-admins,chair,sec";
+  "security" = "root";
+  "ftp" = "root";
+  "ftp-bugs" = "ftp";
+  "hostmaster" = "root";
+  "nagios" = "root";
+  "_nagios" = "root";
+  "ossec" = "root";
+
+# Mailman
+#
+  "mailman" = "root";
+  "mailman-owner" = "mailman";
+  "mailman-request" = "mailman";
+  "mailman-bounces" = "mailman";
+  "mailman-admin" = "mailman";
+
+# Where root mail goes. VERY IMPORTANT!
+#
+  "root" = "rb-admins";
+
+#bit-bucket:		/dev/null
+#dev-null:		bit-bucket
+
+
+#----------------#
+# Administrators #
+#----------------#
+
+# Who wants to get system reports, cron job output etc.
+#
+  "system-reports" = "rb-admins";
+  "audit_warn" = "system-reports";
+
+# Offical way to contact admins for requests.
+#
+  "admin-request" = "rb-admins, ticket";
+  "elected-admin" = "elected-admins";
+
+# Where mail addressed to generic 'admins' goes.
+#
+  "admin" = "admin-request";
+  "admins" = "admin-request";
+
+# DCU admin list.
+#
+# <plop>: Thu May 28 11:07:05 BST 1998
+#
+  "dcu-admin-list" = "rb-admins, sysops@dcu.ie, mcgorman@compapp.dcu.ie";
+
+
+#-------------#
+# Web related #
+#-------------#
+
+# webmaster is now a mailman list
+
+  "httpd" = "webmaster";
+  "www" = "webmaster";
+
+
+#---------------------#
+# Committee & Society #
+#---------------------#
+
+
+# "The Founders" (TM)
+#
+  "founders" = "drjolt, wibble, sandman, fergus, swipe, hyper";
+
+# Committee is now a mailing list (handled by mailman)
+#
+
+# HELP! requests.
+#
+  "help" = "helpdesk";
+  "support" = "helpdesk";
+  "help-request" = "helpdesk";
+  "helpdesk-request" = "helpdesk";
+
+# Chairperson alias.
+#
+  "chairperson" = "chair";
+
+# Treasurer alias.
+#
+  "treasurer" = "treasure";
+
+# Secretary alias.
+#
+  "secretary" = "sec";
+
+# PRO alias.
+#
+# Ents alias.
+#
+  "ents" = "events";
+  "birthday" = "events";
+
+# Accounts alias: - too much shit going to committee -mark
+#
+  "accounts" = "elected-admins, treasurer, chair";
+
+# Redbrick encyclopedia sybmissions/queries.
+#
+# wishkah gave webgroup encylopedia, on conditions he
+# be included in mails about it - <bubble>
+#
+  "encyclopedia" = "wishkah";
+
+# c-hey maintainters alias.
+#
+  "c-hey" = "pooka, colmmacc, bobb";
+
+# DNS.
+#
+# DrJolt Fri Feb 27 12:16:17 GMT 1998
+#
+  "dns" = "committee";
+
+# cancel-announce
+#
+# Cthulhu Thu Jan 27 01:00:00 GMT 2000
+#
+  "cancel-announce" = "bobb";
+
+
+#---------------#
+# Miscellaneous #
+#---------------#
+
+  "wimax" = "johan";
+
+# Alias for STOCS to go with Web Page URL. (Added cthulhu)
+#
+  "sillicon" = "stocs";
+  "spamtastic" = "bubble";
+  "su-webgroup" = "phil, arioch, p, esoteric";
+  "senseigoclub" = "pooka, belial, plop+go";
+  "committee" = "committee@lists.${tld}";
+  "rb-admins" = "rb-admins@lists.${tld}";
```

Comment:
could we group these with the above that alias to them for admins, and committee buisness